tencent/tencent.go

```diff
@@ -125,7 +126,7 @@ func (client Client) Delete(path string) error {
 	if err != nil {
 		return err
 	}
-	req.Header.Set("Host", client.GetEndpoint())
+	req.Header.Set("Host", client.getHost())
```

Comment:
What's the benefit of returning the URL in a fixed format rather than the current `GetEndpoint` implementation?

Actually, could you describe what is the problem and how you fixed it?

Reply:
既然 `client.GetEndpoint()` 可以被用户设置 那看起来已经足以满足要求了?

Reply:
req.Header.Set("Host", client.GetEndpoint()) 不能使用用户设置的域名